index.liquid

```diff
@@ -0,0 +1,199 @@
+      empowers people that publish and use information on the Web. It is a
+      way to create a network of standards-based, machine-readable data across
+      Web sites. It allows an application to start at one piece of Linked Data,
+      and follow embedded links to other pieces of Linked Data that are hosted
+      on different sites across the Web.
+    </p>
+  </div>
+  <div class="eight wide column">
+    <h2 class="ui centered header">A Simple Example</h2>
+    <pre class="ui segment">{
+  "<span style="color: rgb(255, 122, 0);">@context</span>": "<a href="https://json-ld.org/contexts/person.jsonld">https://json-ld.org/contexts/person.jsonld</a>",
+  "<span style="color: rgb(255, 122, 0);">@id</span>": "<a href="http://dbpedia.org/resource/John_Lennon">http://dbpedia.org/resource/John_Lennon</a>",
+  "<span style="color: rgb(255, 122, 0);">name</span>": "<span style="color: rgb(0, 148, 0);">John Lennon</span>",
+  "<span style="color: rgb(255, 122, 0);">born</span>": "<span style="color: rgb(0, 0, 175);">1940-10-09</span>",
+  "<span style="color: rgb(255, 122, 0);">spouse</span>": "<a href="http://dbpedia.org/resource/Cynthia_Lennon">http://dbpedia.org/resource/Cynthia_Lennon</a>"
+}</pre>
+  </div>
+  <div class="justified four wide column">
+    <h2><img class="ui micro inline image" src="images/json-ld-data-32.png" alt="JSON-LD logo"> JSON-LD</h2>
+    <p>JSON-LD is a lightweight Linked Data format. It
+      is easy for humans to read and write. It is based on the already
+      successful JSON format and provides a way to help JSON data interoperate
+      at Web-scale. JSON-LD is an ideal data format for programming
+      environments, REST Web services, and unstructured databases such as
+      Apache CouchDB and MongoDB.
+    </p>
+  </div>
+</div>
+
+<div class="ui large center aligned segment" style="margin: 3em auto">
+  <div class="ui center aligned massive basic segment grid">
+    <div class="ui header">
+      <i class="beer icon"></i>
+      <span class="content">Playground</span>
+    </div>
+  </div>
+  <p class="ui basic huge left aligned padded segment">
+    The <strong>JSON-LD Playground</strong> is a web-based JSON-LD viewer and debugger.
+    If you are interested in learning JSON-LD, this tool will be of great help
+    to you. Developers may also use the tool to debug, visualize, and share
+    their JSON-LD markup.
+  </p>
+  <a href="./playground/" class="ui primary big button">Launch the JSON-LD Playground</a>
+</div>
+
+<div class="ui center aligned massive basic segment grid" id="developers">
+  <div class="ui header">
+    <i class="cog icon"></i>
+    <span class="content">Developers</span>
+  </div>
+</div>
+
+<p class="ui basic large center aligned segment">JSON-LD is available in
+  a number of popular programming environments. Each implementation of JSON-LD
+  listed below is fully conforming to the official JSON-LD specifications.
+</p>
+
+<div class="ui four cards">
```

Comment:
This looks poor on mobile.  Copying a hack from main section.  Perhaps there's a better way.
```suggestion
<div class="ui four cards stackable grid">
```